File: app/src/main/java/github/znzsofficial/adapter/LuaCustRecyclerHolder.java
package github.znzsofficial.adapter;

import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import org.luaj.LuaTable;

public class LuaCustRecyclerHolder extends RecyclerView.ViewHolder {
    public LuaCustRecyclerHolder(@NonNull View itemView) {
        super(itemView);
    }

    public LuaTable Tag = null;

    public void setViews(LuaTable tag) {
        Tag = tag;
    }

    public LuaTable getViews() {
        return Tag;
    }
}

File: app/src/main/java/github/znzsofficial/adapter/LuaCustRecyclerHolder.kt
package github.znzsofficial.adapter

import android.view.View
import androidx.recyclerview.widget.RecyclerView

open class LuaCustRecyclerHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var views: Any? = null

    fun setTag(tag:Any){
        views = tag
    }
    fun getTag() = views
}
